analyzers/tests/SonarAnalyzer.UnitTest/Rules/DisposeNotImplementingDisposeTest.cs

```diff
@@ -35,11 +35,21 @@ public class DisposeNotImplementingDisposeTest
 
         [TestMethod]
         public void DisposeNotImplementingDispose_CSharp9() =>
-            builder.AddPaths("DisposeNotImplementingDispose.CSharp9.Part1.cs", "DisposeNotImplementingDispose.CSharp9.Part2.cs").WithTopLevelStatements().Verify();
+            builder.AddPaths("DisposeNotImplementingDispose.CSharp9.Part1.cs", "DisposeNotImplementingDispose.CSharp9.Part2.cs")
```

Comment:
There's no value in splitting existing UTs that fit on one line into multiple lines.